### ISSUE TITLE
[Accessibility: Resize text] Unable to manage guest interfaces when zoom 200%

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/kimchi#894]
![5e9af195de926030](https://cloud.githubusercontent.com/assets/4921718/14019570/3259a066-f1b2-11e5-9e6a-ee3085381855.jpg)
